docs/reference/index-modules.asciidoc

```diff
@@ -46,6 +46,17 @@ NOTE: The number of shards are limited to `1024` per index. This limitation is a
 
 // end::index-number-of-shards-tag[]
 
+[[index-number-of-routing-shards]]
+`index.number_of_routing_shards`::
+Number of routing shards used to <<indices-split-index,split>> an index and its
```

Comment:
I think the "split an index and its primary shards" sounds a bit confusing. Would ending the sentence after `split an index` makes more sense?